### PR TITLE
net-misc/icaclient: bump to 23.2.0.10; media-plugins/hdx-realtime-media-engine: Bump to version 2.9.600.2900

### PR DESCRIPTION
**Pull request by henning-schild**

### CODE REVIEW

**Comment by henning-schild** [comment]
both tried individually ... no version deps found ... they kind of belong together so one PR

**Comment by henning-schild** [comment]
@juippis @thesamesam please have a look

we have https://bugs.gentoo.org/896180 and need this in the tree to get rid of the two old ones

File: media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.600.2900.ebuild [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+BDEPEND="app-arch/unzip"
+RDEPEND="
+	sys-libs/llvm-libunwind
+	|| (
+		media-video/pipewire
+		media-sound/pulseaudio
+		media-sound/apulse
+	)
+	!<net-misc/icaclient-22.12.0.12
+"
```
Comment (juippis) [inline media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.600.2900.ebuild:29]:
```
 * QA Notice: Unresolved soname dependencies:
 *
 *  /opt/Citrix/ICAClient/rtme/RTMediaEngineSRV: libX11.so.6 libXv.so.1 libpulse.so.0 libsndfile.so.1
 *
```
Please add the providers for these libs into RDEPEND.

Comment (henning-schild) [reply media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.600.2900.ebuild]: I added the libs for the first two, the later two i put on a ignore list like in the parent ebuild

this thing just has multiple implementations and ldd shows stuff that might not all be used/needed

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**